### ISSUE TITLE
[bug]: Unable to use Web UI, gallery block loading infinitely, adding models doesn't work

### ISSUE THREAD

**Issue by odragora** [invoke-ai/InvokeAI#3695]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

16GB

### What version did you experience this issue on?

3.0.0 RC1

### What happened?

When using the web interface, the right panel with the gallery is loading indefinitely.
When you go to the Models -> Add Model, input the path to scan the models for and press the submit button, nothing happens.
Reloading InvokeAI or resetting the Web UI doesn't help.

### Screenshots

<img width="1280" alt="6a2d0991d8d441ae77ac22471621f2bc" src="https://github.com/invoke-ai/InvokeAI/assets/8157117/7286ddf4-b7fe-4316-8885-f83806df1b40">

<img width="1280" alt="395b5ff7be98d89b651100f2869c31c4" src="https://github.com/invoke-ai/InvokeAI/assets/8157117/58810ad7-3ddd-498f-9f43-c870f08d1d85">

### Additional context

While installing InvokeUI, there was an error that stopped the installation, I haven't screenshoted that.
I run the installation the second time into the same folder, and this time the installation finished without errors.

**Comment by odragora** [comment]
I run the number 7 option in the invoke.bat, reinstall / fix the app.
It did install some things it didn't initially, but the gallery was still in the perpetual loading state until I generated the first images and switched back and forth between widget tabs.
The model manager still doesn't work, I can't scan for models and the same error is thrown in the browser console on submit button click.

```
react_devtools_backend_compact.js:2367  Warning: An unhandled error was caught from submitForm() ReferenceError: searchForModels is not defined
    at w (App-9658ed7f.js:88:114130)
    at onSubmit (App-9658ed7f.js:88:115722)
    at App-9658ed7f.js:88:108227
    at App-9658ed7f.js:88:111852
    at App-9658ed7f.js:88:107484
```

**Comment by nelsonre** [comment]
On Mac, I'm also not able to scan for models. The "find models"  button does nothing.

**Comment by psychedelicious** [comment]
Hi @odragora , the gallery problem was fixed in 3.0.0b2 (or maybe it was b3, which is now released).

The model manager is not hooked up yet. I believe this is explained in the release notes.
